public/app/views/team/team.html

```diff
@@ -54,6 +54,26 @@ <h6 class="card-subtitle">Faculty Placement Coordinator</h6>
                             </div>
                         </center>
                     </div>
+
+                    <div class="col-lg-3">
+                        <center class="m-t-30"> <img src="https://res.cloudinary.com/placements-mnit/image/upload/v1615549353/team/gireendra_hjt55y.jpg" class="img-circle" width="150" />
+                            <h4 class="card-title m-t-10">Dr. Gireendra Kumar</h4>
+                            <h6 class="card-subtitle">Faculty Placement Coordinator</h6>
+                            <div class="row text-center justify-content-md-center" ng-show="main.isLoggedIn" ng-show="main.isLoggedIn">
```

Comment:
Please remove `ng-show` tag which has been put twice by mistake, in the old code too. Rest looks GOOD.